app/services/workflow_executions/completion_service.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module WorkflowExecutions
+  # Service used to complete a WorkflowExecution
+  class CompletionService < BaseService
+    def initialize(workflow_execution, user = nil, params = {})
```

Comment:
**nitpick (blocking):** Instead of passing `user` which I don't think will be passed from the `WorkflowExecutionsStatusJob` which is likely where this service will be called from we can just use the `user` from the `WorfklowExecution` which can be accessed via `@workflow_execution.submitter` and passed to `Attachments::CreateService`.

Reply:
fixed in https://github.com/phac-nml/irida-next/pull/439/commits/4aecb110d87bcdc9df22480b43862175567669ad